app/views/freelancers/index.html.erb

```diff
@@ -1,11 +1,17 @@
 <h1>Browse our Techies</h1>
 
-<% @freelancers.each do |freelancer| %>
-<div>
-  <%= image_tag(freelancer.image, size: "100x100", class: "image-fl-index", id: "image_#{freelancer.id}") %>
-  <h3>Name: <%= freelancer.name %></h3>
-  <p><%= freelancer.description %></p>
-  <p>$<%= freelancer.price %>/hr</p>
-  <%= button_to "Choose Me!", cart_index_path(freelancer_id: freelancer.id), class: "btn btn-danger"%>
+<div class="card-deck d-flex justify-content-between">
+  <% @freelancers.in_groups_of(3, false).each do |group| %>
```

Comment:
Nicely done to get the 3 cards at a time. If there's a better way, we can get to it later. :)